cli/utils.js

```diff
@@ -50,16 +50,12 @@ export function getCurrentVersion() {
   return url.match(/@([^/]+)/)?.[1] ?? `local (${url})`;
 }
 
-export function printError(exception, debug = false) {
+export function printError(exception) {
   console.log();
-  console.error(bold(red(`Error:`)), exception.message);
+  console.error(bold(red(`Error:`)), exception);
 
   printDataError(exception);
 
-  if (debug) {
-    console.error(exception);
-  }
```

Comment:
I know this can be repetitive, but I'd print the full exception below the dataError.
I don't know if there's a way to print only the backtrace, to avoid show the message twice.

Reply:
There is a way to just print the stack (`exception.stack`) but it's not standard.

That said, I'm not 100% sure I get what you want the output to be. I updated the function, and now it will output something like (if a preprocessor throws, for example):

```
Error: Error: Error preprocessing page
    at file://.../lume/site.js:482:21
    page: /index.njk

  Error: TypeError: Something awful happened
    at file://..../_config.js:32:9
```

I've also added a try/catch for preprocessor.

Reply:
My idea is display in this order:
- the error message
- the provided variables
- the stack

For example:

```
Error: Error rendering this page
page: /README.md

at file:///Users/oscarotero/test/site.js:497:19

  Error: Couldn't find a loader for this layout
  layout: foo

  at Site.#renderPage (file:///Users/oscarotero/www/lume/lume/site.js:600:15)
  at async file:///Users/oscarotero/test/site.js:494:28
```

Due the stack include also the error message in the first line, maybe something like this:
```js
stack.split("\n").slice(1).forEach((line) => {
    console.log(magenta(`${indent}${line.trim()}`))
});
```